src/utah-design-system/Spinner.jsx

```diff
@@ -0,0 +1,34 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+export default function Spinner({ className }) {
+  return (
+    <svg
+      className={clsx(
+        'h-5 w-5 flex-shrink-0 animate-spin motion-reduce:hidden',
```

Comment:
**question**: what are your thoughts we add some sizes to this, like the button, so we can use this in a button or as part of a page?

Reply:
I had this idea but the `h-*`/`w-*` sizes don't line up with the text sizes that I used for the button. In the end I just elected to allow for the user to override the size via the `className` prop. But I do like the idea of being able to pass in a size that matches the other component. Could be a good future enhancement.

Reply:
wouldn't having the h-5 and w-5 as defaults then add more h-* w-* classes merging with className prop?

Reply:
```jsx
// with
className={clsx(
        'h-5 w-5 flex-shrink-0 animate-spin motion-reduce:hidden',
        className
      )}
/// ...
<Spinner className="h-16 w-16" />
// you would end up with class="h-5 w-5 flex-shrink-0 animate-spin motion-reduce:hidden h-16 w-16"
```
So you have duplicate height and width classes which is not good.

We should make the sizes match with the base font size and the heading sizes. Since you want to add the sizes in a future enhancement, I would suggest removing the h-5 w-5 and require users to set the size when using the component.